packages/roosterjs-editor-plugins/lib/plugins/Paste/wordConverter/wordOnlineTableConverter.ts

```diff
@@ -0,0 +1,24 @@
+import { IEditor } from 'roosterjs-editor-types';
+import { VTable } from 'roosterjs-editor-dom';
+
+/**
+ * @internal
+ * Fix styles of table copied from word online
+ * @param editor
+ * @param table copied from word online
+ */
+export default function wordOnlineTableConverter(editor: IEditor, table: HTMLTableElement) {
+    const vTable = new VTable(table as HTMLTableElement);
+    vTable.cells.forEach(row =>
+        row.forEach(cell => {
+            if (cell.td) {
+                const styledElement = cell.td.querySelector('p');
+                if (editor.isDarkMode()) {
```

Comment:
This should be moved out to the top, if editor is not in dark mode, no need to do all the actions, right? Or do we always want to remove this style?

Reply:
I feel we should always remove it. Otherwise later when user switch to dark mode, there can still be problem.